Sources/NIOPosix/Bootstrap.swift

```diff
@@ -712,8 +712,17 @@ private extension Channel {
 
 /// A `ClientBootstrap` is an easy way to bootstrap a `SocketChannel` when creating network clients.
 ///
-/// Usually you re-use a `ClientBootstrap` once you set it up and called `connect` multiple times on it.
-/// This way you ensure that the same `EventLoop`s will be shared across all your connections.
+/// You may re-use a `ClientBootstrap` once you set it up and call `connect` multiple times on it.
+/// This ensures all connections you create share the same `EventLoop`.
```

Comment:
This is not true. We are taking the next `EventLoop` from the `EventLoopGroup` in `connect`.

Reply:
fixed thanks